assemblies/working-with-pipelines-in-jupyterlab.adoc

```diff
@@ -7,6 +7,10 @@ ifdef::context[:parent-context: {context}]
 
 include::modules/overview-of-pipelines-in-jupyterlab.adoc[leveloffset=+1]
 
+include::modules/caching-nodes-in-elyra.adoc[leveloffset=+1]
+
+include::modules/disable-node-caching-in-elyra.adoc[leveloffset=+2]
```

Comment:
Move to after the " include::modules/updating-a-runtime-configuration.adoc[leveloffset=+1]" line

I'm not certain this is the correct spot - but it seems like they are updating a runtime configuration by editing the nodes.